src/server/nodeimpl.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference path="node.d.ts" />
+module ts.server {
+    export class NodeEnvironment implements Environment {
+        byteLength = Buffer.byteLength;

```

Comment:
That's pretty much what this is doing, it was just bundling the two parameters into one argument. I'll split it out into multiple arguments.